app/sider/Dokumentasjon.tsx

```diff
@@ -1,34 +1,77 @@
-import { Button } from '@navikt/ds-react';
-import { useNavigate } from '@remix-run/react';
+import { Alert, Button } from '@navikt/ds-react';
+import { useActionData, useNavigate, useSubmit } from '@remix-run/react';
+import { useEffect, useState } from 'react';
 
-import { useTekster, useYtelse } from '~/hooks/contextHooks';
+import {
+  useEndringsmelding,
+  useEndringsmeldingMottattDato,
+  useTekster,
+  useYtelse,
+} from '~/hooks/contextHooks';
 import HovedInnhold from '~/komponenter/hovedInnhold/HovedInnhold';
 import StegIndikator from '~/komponenter/stegindikator/StegIndikator';
 import TekstBlokk from '~/komponenter/tekstblokk/TekstBlokk';
+import { action as actionBA } from '~/routes/ba.dokumentasjon';
+import { action as actionKS } from '~/routes/ks.dokumentasjon';
 import { ESanityMappe, ESteg } from '~/typer/felles';
+import { EFritekstFeil, fritekstFeilTilApiKeys } from '~/typer/fritekstfeil';
+import { EStatusKode, IPostResponse } from '~/typer/response';
 import { ETypografiTyper } from '~/typer/typografi';
+import { EYtelse } from '~/typer/ytelse';
 import { hentPathForSteg } from '~/utils/hentPath';
 
 import css from './dokumentasjon.module.css';
 
 export default function DokumentasjonSide() {
   const ytelse = useYtelse();
+  const teksterDokumentasjon = useTekster(ESanityMappe.DOKUMENTASJON);
+  const teksterFelles = useTekster(ESanityMappe.FELLES);
+  const teksterSendEndringer = useTekster(ESanityMappe.SEND_ENDRINGER);
+  const [endringsmelding] = useEndringsmelding();
+  const [, settEndringsmeldingMottattDato] = useEndringsmeldingMottattDato();
+
   const navigate = useNavigate();
+  const submit = useSubmit();
 
-  const tekster = useTekster(ESanityMappe.DOKUMENTASJON);
-  const teksterFelles = useTekster(ESanityMappe.FELLES);
+  const action = ytelse === EYtelse.BARNETRYGD ? actionBA : actionKS;
```

Comment:
Kan dette flyttes til en utils-funksjon? Typ `hentAction(ytelse)` med switch case på ytelser. Da legger vi bedre opp for å ta i mot flere stønader senere 😁

Reply:
https://github.com/bekk/nav-familie-endringsmelding/pull/119/commits/220e39b87c7db108d2ef8d9cd80efbd6d30dbb4a 🤖